Sources/MapboxDirections/RestStop.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+/**
+ `RestStop` describes corresponding object on the route.
+ */
+public struct RestStop: Codable, Equatable {
+
+    public enum StopType: String, Codable {
+        case serviceArea = "service_area"
+        case restArea = "rest_area"
+    }
+
+    let type: StopType
```

Comment:
This property is internal, so clients such as the navigation SDK don’t have access to it.